Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM registry.tld/ruby-fips-base-image-ubuntu:1.0.0
```

Comment:
Is this a private Docker repo URL?  I thought that we were going to publish this so that it's publicly available.

Reply:
That is a really good point! I had missed that this in particular would prevent anyone from build the Conjur image without internal registry access.

Reply:
@nahumcohen what is needed to publish the image to our dockerhub? if you need help getting the base image repo reviewed to make it public, and adding a dockerhub to also push this image to, please let me know. I agree with @alexkalish, it is very important that we maintain the ability of community members to build this project locally.

Reply:
This is only WIP PR, next step is to publish base image to dockerhub.
Does the image require any license?

Reply:
@nahumcohen Just an FYI: when creating a PR, you can choose to open a "Draft PR" (use arrow dropdown on the GH button).

Reply:
@nahumcohen can we at least put a WIP in the PR title until it's ready for review? Since it's not a draft and doesn't have this heading, there's no indication that this PR is in a pre-mergeable state.

Reply:
Is there a ticket for making the base image publicly available?  Without it, we're preventing anyone from outside our org from working on Conjur.

Reply:
Done